src/main/resources/application.properties

```diff
@@ -9,15 +9,15 @@
 
 # Determines the type of credential used for Key Vault Access
 # Valid Values = MSI || CLI
-# Linux: export He_EnvironmentFlag=CLI
+
 # Windows: set He_EnvironmentFlag=CLI
 helium.environment.flag = ${He_EnvironmentFlag}
 
 # If helium.environment.flag == CLI and you need a custom path
 # to the Azure profile and tokens then set the following two values.
 # Otherwise they will be ignored.
-azure.profile.path=
-azure.accessTokens.path=
+azure.profile.path=Users\\jyraviku\\.azure\\azureProfile.json
```

Comment:
Don't checkin this change with dev specific attributes

Reply:
Done!